src/components/PrimaryPanes/SourcesTreeItem.js

```diff
@@ -150,21 +150,23 @@ class SourceTreeItem extends Component<Props, State> {
   }
 
   renderItemName() {
-    const { item, hasMatchingGeneratedSource } = this.props;
-    const suffix = hasMatchingGeneratedSource ? " [sm]" : "";
+    const { item } = this.props;
 
     switch (item.name) {
       case "ng://":
         return "Angular";
       case "webpack://":
         return "Webpack";
       default:
-        return `${item.name}${suffix}`;
+        return `${item.name}`;
     }
   }
 
   render() {
-    const { item, depth, focused } = this.props;
+    const { item, depth, focused, hasMatchingGeneratedSource } = this.props;
+    const suffix = hasMatchingGeneratedSource ? (
```

Comment:
We're going to need a follow up PR for this which creates/moves `getSuffixForSource()` to a source util so I can use it in other places, like tabs and breakpoint sidebar, but I'll do that with tabs work.